### DISCUSSION TITLE
Dreamcast not powering ESP32

### DISCUSSION THREAD

**Discussion by s3anami** [darthcloud/BlueRetro#287] [Q&A]
I have all 4 cords wired as diagrammed here:  
https://imgur.com/a/fSgehIu

I have redone the soldering a few times and doesn't seem to make a difference

It is a china clone with the weird cap soldered on diagonally, not sure if that breaks 5 v?

**Comment by darthcloud** [comment]
Try getting official devkitc ESP32-DEVKITC-32E with ESP-WROOM-32 (Get it on [Digikey](https://www.digikey.com/en/products/detail/espressif-systems/ESP32-DEVKITC-32E/12091810?s=N4IgTCBcDaIKYGcAOBmMBaAJnAbgawEsAXAY3TThAF0BfIA) or [Mouser](https://www.mouser.com/ProductDetail/Espressif-Systems/ESP32-DevKitC-32E?qs=sGAEpiMZZMv0NwlthflBi3XYxq%252BTapwyHnR2%2FrAf4bY%3D))

**Comment by s3anami** [comment]
i got a different esp32 module and it still won't work, its the only system I can't seem to get working that I tried. Did N64 and Gamecube ones so far

**Reply by darthcloud** [reply]
Disconnect 5V pin from Dreamcast and power from USB using laptop.

Try to get the log while doing so:
https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10
